src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,122 @@
+       [:<>
+        [rn/view {:style {:flex-direction :row}}
+         [quo2/text
+          {:weight :medium
+           :size   :paragraph-1
+           :style  (style/account-name (colors/theme-colors colors/neutral-100 colors/white theme))}
+          account-name]
+         (when (= account :watched-address)
+           [icons/icon :i/reveal
+            {:container-style style/reveal-icon
+             :color           (colors/theme-colors colors/neutral-80-opa-40
+                                                   colors/white-opa-40
+                                                   theme)}])]
+        [quo2/text
+         {:weight :semi-bold
+          :size   :heading-1
+          :style  style/current-value}
+         current-value]
+        [rn/view
+         {:style style/row-centered}
+         [:<>
+          [quo2/text
+           {:weight :medium
+            :size   :paragraph-2
+            :style  (style/bottom-time-text (and (not= :custom time-frame)
+                                                 (seq time-frame-to-string)))}
+           (when (seq time-frame-string)
+             time-frame-string)]
+          (when (and (= :custom time-frame)
+                     (seq time-frame-to-string))
+            [:<>
+             [icons/icon :i/positive-right
+              {:size            16
+               :container-style style/right-arrow}]
+             [quo2/text
+              {:weight :medium
+               :size   :paragraph-2
+               :style  style/bottom-time-to-text}
+              time-frame-to-string]])]
+         (when (and (seq percentage-change)
+                    (seq currency-change))
+           [:<>
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/percentage-change (colors/theme-colors
+                                                (colors/custom-color customization-color 50)
+                                                (colors/custom-color customization-color 60)
+                                                theme))}
+             percentage-change]
+            [rn/view
+             {:style (style/dot-separator (colors/theme-colors
+                                           (colors/custom-color customization-color 60 40)
+                                           (colors/custom-color customization-color 50 40)
+                                           theme))}]
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/currency-change (colors/theme-colors
+                                              (colors/custom-color customization-color 50)
+                                              (colors/custom-color customization-color 60)
+                                              theme))}
+             currency-change]
+            [icons/icon
+             (if up?
+               :i/positive
+               :i/negative)
```

Comment:
@ibrkhalil The component tree is too big. Better break it up into smaller components.

@ilmotta @J-Son89 What do you guys think of adding a tree `max-lines-length` to our guidelines? (or is it possible to be added to linter?)

Reply:
I agree that probably it's nice to break this into smaller pieces but so sure we need a fixed rule for this. 
No strong opinions but I think we all have a good enough sense of when something can be broken down smaller.

Reply:
There's a lot of fun stuff [here](https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md)

Reply:
There are reasons this kind of rule is usually not implemented/used in React projects. There's a spectrum of tree sizes that are acceptable and that eventually become too big. I agree with @J-Son89, some things are better if we create awareness in the team, since there's no definite tree size rule, it's more about React best practices.